docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst

```diff
@@ -271,7 +290,7 @@ data_dir = J:/Project/simulation_science/multiple_myeloma/data/cause_model_input
    * - incidence_MM
      - S
      - NDMM
-     - :math:`\frac{\text{incidence_c486}}{1-\text{prev_c486}}`
+     - :math:`\frac{\text{incidence_c486} * 50}{1 - \text{prev_c486}}`
```

Comment:
```suggestion
     - :math:`\frac{\text{incidence_c486} * 50}{1 - \text{prev_c486}*50}`
```

I think

Reply:
See my note above -- I considered this. If my math is right (it's probably not), this makes our bias depend on the difference between sim prevalence and GBD prevalence, so it's hard to say definitively that it decreases the bias.

Reply:
Is sim prevalence off from GBD by a factor of 50? We might be able to make a decision about what's "better" here by magnitude of differences

Reply:
Sorry -- when I say "difference between sim prevalence and GBD prevalence" I mean the difference _after_ accounting for a 50x multiplier.

In our Phase 1 results (no multiplier), we substantially overestimate MM prevalence relative to GBD in some age groups. That would create a downward bias in incidence per prevalent case, whereas Abie's proposed change could only increase incidence. But, if we ever had a prevalence _underestimate_ in our sim, the biases could compound. Without knowing in advance how big of an underestimate would be possible, I don't know how to tell whether this change would be better or worse.

Also, if we do this I will need some help with the math on what multiplier to choose, I tried working it out and got stuck 😅

Reply:
IIUC, the Phase I prevalence in the sim was different than the GBD prevalence because we did not use the GBD mortality rate in Phase I.  So transforming the population incidence rate to the without-condition incidence rate has always been a little off.  If we multiply everything by 50, this discrepancy might matter more than it did in Phase I, since the GBD prevalence is so low that the population and without-condition incidence rates are almost identical.

So maybe the compromise solution is to multiply everything by 5 or 10 instead of 50.

Reply:
I'm having difficulty understanding everything well enough to give useful feedback, but I'd be happy to chat with you, @zmbc , so that I can ask questions to help me figure it out.

One question I have now: How did you derive the above formula for the "bias relative to the true GBD incidence count per prevalent case"?